Dockerfile.agnos

```diff
@@ -78,25 +78,30 @@ RUN /tmp/agnos/openpilot_python_dependencies.sh
 COPY --from=agnos-compiler-ffmpeg /tmp/ffmpeg.deb /tmp/ffmpeg.deb
 RUN cd /tmp && apt-get -o Dpkg::Options::="--force-overwrite" install -yq ./ffmpeg.deb
 
-RUN export PATH="/usr/local/pyenv/bin:/usr/local/pyenv/shims:$PATH" && \
-    export PYENV_ROOT="/usr/local/pyenv" && \
-    eval "$(pyenv init -)" && \
-    eval "$(pyenv virtualenv-init -)" && \
+RUN export PATH="$HOME/.cargo/bin:$PATH" && \
+    export UV_BIN="$HOME/.cargo/bin" && \
     export MAKEFLAGS="-j$(nproc)" && \
+    source .venv/bin/activate && \
     pip install pyqt5-sip==12.12.1 && \
-    pip install pyqt5==5.15.9 --verbose --config-settings --confirm-license= && \
-    pyenv rehash
+    pip install pyqt5==5.15.9 --verbose --config-settings --confirm-license= 
 
 # Install openpilot python packages
 RUN mkdir -p /tmp/agnos/uv
 COPY ./userspace/pyproject.toml ./userspace/uv.lock /tmp/agnos/uv
-RUN export PATH="/usr/local/pyenv/bin:/usr/local/pyenv/shims:$PATH" && \
-    export PYENV_ROOT="/usr/local/pyenv" && \
-    eval "$(pyenv init -)" && \
-    eval "$(pyenv virtualenv-init -)" && \
+RUN export PATH="$HOME/.cargo/bin:$PATH" && \
+    export UV_BIN="$HOME/.cargo/bin" && \
+    source .venv/bin/activate && \
     cd /tmp/agnos/uv && \
     export PYOPENCL_CL_PRETEND_VERSION="2.0" && \
-    MAKEFLAGS="-j$(nproc)" UV_NO_CACHE=1 uv pip install --no-cache-dir --system .
+    
```

Comment:
Remove empty line to prevent the following docker warning:
WARN: NoEmptyContinuation: Empty continuation line